adapters/audienceNetwork/audienceNetworktest/supplemental/server-error-500.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
It looks like we're missing test coverage for the the error handling block on line 368 of facebook.go where we check that the stringified json `adm` is well-formed: 
```
var obj facebookAdMarkup
if err := json.Unmarshal([]byte(bid.AdM), &obj); err != nil {
	errs = append(errs, &errortypes.BadServerResponse{
		Message: err.Error(),
	})
	continue
}
```
Perhaps you want to punt on this but I see you added coverage for the other error paths so I figured I'd mention it.

Reply:
Thanks for calling it out. I've added a test for this case, but I'm fine cutting off coverage expansion for this PR. It's also not possible to test `json.Marshal` failures in most cases without extracting and mocking the json package.